### DISCUSSION TITLE
Can't see rsyslog logs in Security Onion Server

### DISCUSSION THREAD

**Discussion by ivanrdrgz** [Security-Onion-Solutions/securityonion#12029] [Unsupported Versions]
Hi everyone,

I have a deployment with Security Onion 2.3.230 (2.4 does not fit my scenario), Standalone architecture , and I am trying to send logs from Ubuntu systems to my Security Onion server and be able to see alerts related to activity in Ubuntu endpoints. I already so-allow the endpoints and configure them to send the logs to 192.168.30.10 (my Security Onion instance) with rsyslog, but I can't see any activity. 

I don't quite understand how Security Onion treats these logs and how I should be able to view them through Kibana or the Security Onion Console.

Who should be listening in port 514 in Security Onion when I make so-allow?

Thank you so much for your help in advance,
Regards

**Comment by dougburks** [comment]
If you run tcpdump on the management interface of your Security Onion instance, do you see the syslog traffic?

Please keep in mind that 2.3 reaches End Of Life in less than 4 months, so you should start planning your migration to 2.4 soon:
https://blog.securityonion.net/2023/12/4-month-end-of-life-eol-reminder-for.html
